### DISCUSSION TITLE
Thoughts on adding Sponsorship?

### DISCUSSION THREAD

**Discussion by daniel-xyz** [react-native-maps/react-native-maps#5266] [Q&A] [answered]
Hey everyone,

First off, big thanks to the maintainers for all the work you’ve put into react-native-maps! :)

I was wondering if you’d consider enabling sponsorship (like GitHub Sponsors or similar)? I think a lot of folks, myself included, would love the chance to chip in if it could help someone passionate about the project dedicate more time to it and to show gratitude. It's such an essential library for many apps out there.

Just wanted to throw the idea out there and see what you think.

**Comment by salah-ghanim** [comment, marked as answer]
@daniel-xyz I just added sponsorship to my own profile, yes I agree completely if a small group of companies pitched in this library can be in a much better state.

**Reply by daniel-xyz** [reply]
Great! 😃 I've kicked things off with the first sponsorship and will increase the amount as my own project grows. Let's see if others follow. Perhaps it should also be mentioned in the repository's README.

**Reply by salah-ghanim** [reply]
@daniel-xyz I'm slowly starting this, thank you so much for doing so, yes I agree with more sponsors this library can be professionally maintained and at least stay up to date with native and react-native changes
